internal/dockercompose/client.go

```diff
@@ -128,7 +128,7 @@ func (c *cmdDCClient) Up(ctx context.Context, spec v1alpha1.DockerComposeService
 	c.mu.Lock()
 	defer c.mu.Unlock()
 	runArgs := append([]string{}, genArgs...)
-	runArgs = append(runArgs, "up", "--no-deps")
+	runArgs = append(runArgs, "up", "--no-deps", "--remove-orphans")
 	// Omit --no-build for now to get v2 working.
 	// https://github.com/docker/compose/issues/8785
```

Comment:
I wonder if this can be cleaned up now too?

Reply:
ooh good point! done